db/schema.rb

```diff
@@ -316,9 +316,9 @@
     t.integer "failed_attempts", default: 0
     t.string "unlock_token"
     t.datetime "locked_at", precision: nil
+    t.string "phone"
```

Comment:
Let's remove this since there's no migration here

Reply:
Hm, this happens every time I migrate, I assume that it does to other people too? Wouldn't it be easier to commit this instead of unstaging this file every time we do PRs? Or do you mean just not in this PR?

Reply:
I think this happens every time because migrations have been run out of order on your local (i.e. you've run a migration while working on a branch, not rolled it back, and then run one that was merged into main before merging your PR).

If you dropped your DB and ran migrations on a fresh one I don't think you'd get this (if you did we should commit it)

Reply:
Hm, so after running `db:drop` and `db:reset` my schema looks the way it does here. I'll commit this!